Feat: Add meeting minutes 2026-04-29 (#1459)
oasis-tcs/csaf: meeting_minutes/2026/2026-04-29.md

![image](https://user-images.githubusercontent.com/1690898/139102180-5c1e2583-14f1-4f58-ab2b-9e3807ed529c.png)

# Common Security Advisory Framework (CSAF) Technical Committee Working Meeting

- Meeting Date: April 29, 2026
- Time: 18:00 UTC (20:00 CET, 13:00 EST, 10:00 PST)

## Call to Order and Welcome

- Meeting called to order @ 18:05 UTC.

## Roll call

All participants recorded their attendance on the OQC system.

## Participants

> Quorum requires participation of 5 or more of the 9 (10 - 1) voting members (including the officers).

Reviewer: ```suggestion
Quorum requires participation of 6 or more of the 11 (12 - 1) voting members (including the officers).
```

| Given Name | Family Name | Affiliation                                                 | Role             | Present |

> |:-----------|:------------|:------------------------------------------------------------|:-----------------|:--------|

Reviewer: Missing Celia here

```suggestion
|:-----------|:------------|:------------------------------------------------------------|:-----------------|:--------|
| Celia      | Savidge     | Dell                                                        | Voting Member         | Yes     |
```

Author: Thank You! Addressed.

> | Christian  | Banse       | Fraunhofer-Gesellschaft e.V. - Fraunhofer AISEC             | Voting Member    |         |

Reviewer: ```suggestion
| Christian  | Banse       | Fraunhofer-Gesellschaft e.V. - Fraunhofer AISEC             | Voting Member    |  No       |
```

| Christoph  | Plutte      | Ericsson AB                                                 | Voting Member    | Yes     |

> | Dina       | Truxius     | Federal Office for Information Security (BSI)               | Member           |         |

Reviewer: ```suggestion
| Dina       | Truxius     | Federal Office for Information Security (BSI)               | Member           |   No      |
```

> | Feng       | Cao         | Oracle                                                      | Voting Member    |         |

Reviewer: ```suggestion
| Feng       | Cao         | Oracle                                                      | Voting Member    | No        |
```

| JD         | Stefaniak   | Dell                                                        | Voting Member    | Yes     |

> | Jane       | Ginn        | Cyber Threat Intelligence Network, Inc.                     | Member           |         |

Reviewer: ```suggestion
| Jane       | Ginn        | Cyber Threat Intelligence Network, Inc.                     | Member           | No        |
```

> | Jessie     | Vaught      | Red Hat                                                     | Voting Member    |         |

Reviewer: ```suggestion
| Jessie     | Vaught      | Red Hat                                                     | Voting Member    | No        |
```

> | Johannes   | Lorenz      | Siemens AG                                                  | Member           |         |

Reviewer: ```suggestion
| Johannes   | Lorenz      | Siemens AG                                                  | Member           | No        |
```

> | Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Leave of Absence |         |

Reviewer: ```suggestion
| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Leave of Absence |  No       |
```

> | Michael    | Reeder      | Dell                                                        | Member           |         |

Reviewer: ```suggestion
| Michael    | Reeder      | Dell                                                        | Member           | No        |
```

> | Michael    | Schueler    | Cisco Systems                                               | Voting Member    |         |

Reviewer: ```suggestion
| Michael    | Schueler    | Cisco Systems                                               | Member           | No        |
```

| Omar       | Santos      | Cisco Systems                                               | Co-Chair         | Yes     |
| Sergii     | Demianchuk  | Cisco Systems                                               | Voting Member    | Yes     |

> | Celia      | Savidge     |                                                             | Voting Member    | Yes     |

Reviewer: We are sorting by first name - I added her on top
```suggestion
```

| Sonny      | van Lingen  | Huawei Technologies Co., Ltd.                               | Voting Member    | Yes     |
| Stefan     | Hagen       | Individual                                                  | Co-Chair         | Yes     |

> | Thomas     | Proell      | Siemens AG                                                  | Member           |         |

Reviewer: ```suggestion
| Thomas     | Proell      | Siemens AG                                                  | Member           | No        |
```

> | Thomas     | Schaffer    | Cisco Systems                                               | Member           |         |

Reviewer: ```suggestion
| Thomas     | Schaffer    | Cisco Systems                                               | Member           | No        |
```

| Thomas     | Schmidt     | Federal Office for Information Security (BSI)               | Voting Member    | Yes     |
| Tobias     | Limmer      | Siemens AG                                                  | Voting Member    | Yes     |
| Vijay      | Sarvepalli  | SEI                                                         | Member           |         |
| Vivek      | Nair        | Microsoft Corporation                                       | Member           |         |

### Observers present

Note: Observers of this committee that are ready to become Members should follow the specific instructions displayed the OASIS Open Notices tab.

### Quorum

- Quorum was **reached**.

## Agenda

- Roll Call
- Updates or news to the TC
- Review all open GitHub issues and pull requests requiring TC discussion.
- Next steps
- Adjourn

## Identification of TC voting members (Secretary)

### Prospective voting members attending their first meeting

### Members attaining voting rights at the end of this meeting

### Members losing voting rights if they have not joined this meeting by the time it ends

> 

Reviewer: ```suggestion

- Christian Banse
- Jessie Vaught

```

Author: Thank You! Addressed.

### Members who previously lost voting rights who are attending this meeting

> 

Reviewer: ```suggestion

- Omar Santos

```

Author: Thank You! Addressed.

![image](https://user-images.githubusercontent.com/1690898/139102180-5c1e2583-14f1-4f58-ab2b-9e3807ed529c.png)

# Common Security Advisory Framework (CSAF) Technical Committee Working Meeting

- Meeting Date: April 29, 2026

> - Time: 18:00 UTC (20:00 CET, 13:00 EST, 10:00 PST)

Reviewer: ```suggestion
- Time: 17:00 UTC (19:00 CEST, 13:00 EDT, 10:00 PDT)
```

Author: Thank You! Comments applied and resolved.

## Call to Order and Welcome

> - Meeting called to order @ 18:05 UTC.

Reviewer: ```suggestion
- Meeting called to order @ 17:05 UTC.
```

Author: Thank You! Comments applied and resolved.

| Michael    | Schueler    | Cisco Systems                                               | Member           | No      |
| Omar       | Santos      | Cisco Systems                                               | Co-Chair         | Yes     |
| Sergii     | Demianchuk  | Cisco Systems                                               | Voting Member    | Yes     |
| Sonny      | van Lingen  | Huawei Technologies Co., Ltd.                               | Voting Member    | Yes     |
| Stefan     | Hagen       | Individual                                                  | Co-Chair         | Yes     |
| Thomas     | Proell      | Siemens AG                                                  | Member           | No      |
| Thomas     | Schaffer    | Cisco Systems                                               | Member           | No      |
| Thomas     | Schmidt     | Federal Office for Information Security (BSI)               | Voting Member    | Yes     |
| Tobias     | Limmer      | Siemens AG                                                  | Voting Member    | Yes     |
| Vijay      | Sarvepalli  | SEI                                                         | Member           |         |

> | Vivek      | Nair        | Microsoft Corporation                                       | Member           |         |

Reviewer: ```suggestion
| Vijay      | Sarvepalli  | SEI                                                         | Member           | No      |
| Vivek      | Nair        | Microsoft Corporation                                       | Member           | No      |
```

Author: Thank You! Comments applied and resolved.

### Observers present

Note: Observers of this committee that are ready to become Members should follow the specific instructions displayed the OASIS Open Notices tab.

### Quorum

- Quorum was **reached**.

## Agenda

- Roll Call
- Updates or news to the TC
- Review all open GitHub issues and pull requests requiring TC discussion.
- Next steps
- Adjourn

## Identification of TC voting members (Secretary)

### Prospective voting members attending their first meeting

### Members attaining voting rights at the end of this meeting

### Members losing voting rights if they have not joined this meeting by the time it ends

- Christian Banse
- Jessie Vaught

### Members who previously lost voting rights who are attending this meeting

- Omar Santos

### Members who have declared a leave of absence

- Justin Murphy

## Meeting Notes

### Leave of Absence

> - A motion was presented by Thomas Schmidt to accept Justin Murphy's leave of absence. No objections were raised. The motion carries.

Reviewer: ```suggestion
- A motion was presented by Thomas Schmidt to accept Justin Murphy's leave of absence, seconded by Sergii Demianchuk. No objections were raised. The motion carries.
```

Author: Thank You! Comments applied and resolved.

- PR [#1401](https://github.com/oasis-tcs/csaf/pull/1401) - Adds a comment explaining the change from open-ended model numbers, serial numbers and SKUs to the fixed-length variant in CSAF 2.1 and why a conversion is needed.
- PR [#1402](https://github.com/oasis-tcs/csaf/pull/1402) - Formatting fixes and clarification regarding the conversion of TLP labels in the distribution text; replaces unclear wording with clearer text.
- PR [#1404](https://github.com/oasis-tcs/csaf/pull/1404) - Clarifies that converters must emit a warning when information cannot be unambiguously produced, and adds that multiple license expressions found in the same variant must also raise a warning.
- PR [#1406](https://github.com/oasis-tcs/csaf/pull/1406) - Clarifies Test 6.2.38 (detection of usage of deprecated profiles); adds implementation advice and test files to help implementers verify their implementation.
- PR [#1407](https://github.com/oasis-tcs/csaf/pull/1407) - Clarifies Test 6.1.54 and adds test files for implementers.
- PR [#1408](https://github.com/oasis-tcs/csaf/pull/1408) - Fixes a typo.

> - PR [#1422](https://github.com/oasis-tcs/csaf/pull/1422) - Adds missing test files requested by Sonny van Lingen.

Reviewer: ```suggestion
- PR [#1422](https://github.com/oasis-tcs/csaf/pull/1422) - Adds missing test files requested by implementers.
```

Author: Thank You! Suggestions were applied and comment is resolved.

- A motion was presented by Thomas Schmidt to accept PR #1412 into CSAF 2.1 and seconded by Sonny van Lingen. No objections, discussions or concerns were raised. The motion carries.

#### PR [#1420](https://github.com/oasis-tcs/csaf/pull/1420) - Documentation of TC standing rules

- Thomas Schmidt presented PR #1420, which documents the standing rules already taken from the meeting minutes and adds them as a separate folder so the TC can keep track of agreed standing rules. The email-motion standing rule (originating very early in the TC's history) is not yet included and can be added later.
- Sonny van Lingen asked for confirmation that this is purely an administrative listing of existing standing rules and does not introduce any new ones. Thomas Schmidt confirmed.
- A motion was presented by Thomas Schmidt to accept the documentation of the TC's standing rules into the CSAF repository as stated in PR #1420 and seconded by Christoph Plutte. No objections, discussions or concerns were raised. The motion carries.

#### PR [#1427](https://github.com/oasis-tcs/csaf/pull/1427) - Registry for Vulnerability ID Systems (RVIDS) and governance standing rule

> - Thomas Schmidt presented PR #1427, which originated from a comment by Sonny van Lingen during the public comment period requesting clear guidance for the RVIDS. The PR contains:

Reviewer: ```suggestion
- Thomas Schmidt presented PR #1427, which originated from a comment by Sonny van Lingen during the public comment period requesting clear guidance for the RVISC. The PR contains:
```

Author: Thank You! Suggestions were applied and comment is resolved.

>   - A description of what RVIDS is and how it works (standardized names for vulnerability ID systems used in the `system_name` / `text` properties, with text patterns to validate entries).

Reviewer: ```suggestion
  - A description of what RVISC is and how it works (standardized names for vulnerability ID systems used in the `system_name` / `text` properties, with text patterns to validate entries).
```

Author: Thank You! Suggestions were applied and comment is resolved.

  - A mapping mechanism from currently used values to registered values, with the algorithm describing how mappings are produced.
  - A governance and registration process (under the "Changes" section) which is intended to become a standing rule.

> - Sonny van Lingen asked whether new items added to the mapping would bypass a TC vote. Thomas Schmidt clarified that only new mapping entries (when the ID system is already in RVIDS, evidence exists in real-world files, and editors confirm no negative impact on other entries) are added by the change controller and approved by editors. New ID systems still require a TC vote.

Reviewer: ```suggestion
- Sonny van Lingen asked whether new items added to the mapping would bypass a TC vote. Thomas Schmidt clarified that only new mapping entries (when the ID system is already in RVISC, evidence exists in real-world files, and editors confirm no negative impact on other entries) are added by the change controller and approved by editors. New ID systems still require a TC vote.
```

Author: Thank You! Suggestions were applied and comment is resolved.

> - Sergii Demianchuk asked whether RVIDS is a response to the EU Cyber Resilience Act. Thomas Schmidt explained the motivation is broader: CVE remains a first-class citizen, but many vendor-specific IDs (e.g. Cisco bug IDs, Red Hat Bugzilla IDs, GitHub issues) are named differently across the supply chain. RVIDS provides authoritative system names to correlate advisories that refer to the same vulnerability. Discussion followed about how this helps tracking across the supply chain (e.g. Log4Shell, where vendors initially used internal bug IDs before a CVE was assigned).

Reviewer: ```suggestion
- Sergii Demianchuk asked whether RVISC is a response to the EU Cyber Resilience Act. Thomas Schmidt explained the motivation is broader: CVE remains a first-class citizen, but many vendor-specific IDs (e.g. Cisco bug IDs, Red Hat Bugzilla IDs, GitHub issues) are named differently across the supply chain. RVISC provides authoritative system names to correlate advisories that refer to the same vulnerability. Discussion followed about how this helps tracking across the supply chain (e.g. Log4Shell, where vendors initially used internal bug IDs before a CVE was assigned).
```

Author: Thank You! Suggestions were applied and comment is resolved.

- Sonny van Lingen noted that mixing this with SBOM linkage goes beyond the intended scope; Thomas Schmidt confirmed this is mainly about identifying the same vulnerability via standardized system names (especially important for VEX where a CVE may not exist).

> - Sergii Demianchuk pointed out that the link in the registry currently redirects to raw GitHub. Thomas Schmidt confirmed the long-term plan is for the registry to be served directly under `registry.csaf.io`.

Reviewer: ```suggestion
- Sergii Demianchuk pointed out that the link in the registry currently redirects to raw GitHub. Thomas Schmidt confirmed the long-term plan is for the registry to be served directly under `registry.csaf.dev`.
```

Author: Thank You! Suggestions were applied and comment is resolved.

- Sonny van Lingen asked whether EUVD should be voted in as a registered system to maintain procedural hygiene. Thomas Schmidt agreed to vote on the existing entries.

A motion was presented by Thomas Schmidt to accept PR #1427 into the CSAF 2.1 repository, with the caveat of not adopting the standing rule suggested in lines 65 and following (which would be voted separately). The motion was seconded by Sergii Demianchuk (also seconded by Christoph Plutte). No objections, discussions or concerns were raised. The motion carries.

> A motion was presented by Thomas Schmidt to accept EUVD as a registered system into RVIDS as stated in the registry. The motion was seconded by Christoph Plutte. No objections, discussions or concerns were raised. The motion carries.

Reviewer: ```suggestion
A motion was presented by Thomas Schmidt to accept EUVD as a registered system into RVISC as stated in the registry. The motion was seconded by Christoph Plutte. No objections, discussions or concerns were raised. The motion carries.
```

Author: Thank You! Suggestions were applied and comment is resolved.

> A motion was presented by Thomas Schmidt to accept the OASIS Open CSAF TC GitHub Issues as a registered system into RVIDS as stated in the registry. The motion was seconded by Christoph Plutte. No objections, discussions or concerns were raised. The motion carries.

Reviewer: ```suggestion
A motion was presented by Thomas Schmidt to accept the OASIS Open CSAF TC GitHub Issues as a registered system into RVISC as stated in the registry. The motion was seconded by Christoph Plutte. No objections, discussions or concerns were raised. The motion carries.
```

Author: Thank You! Suggestions were applied and comment is resolved.

> A motion was presented by Thomas Schmidt to hold a Full Majority Vote on adopting the standing rule suggested in lines 65 and following of PR #1427 (governance process for registering ID systems and mappings in RVIDS). The motion was seconded by Christoph Plutte. No objections, discussions or concerns were raised. Each voting member present then stated their vote. The following Voting Members voted in favor:

Reviewer: ```suggestion
A motion was presented by Thomas Schmidt to hold a Full Majority Vote on adopting the standing rule suggested in lines 65 and following of PR #1427 (governance process for registering ID systems and mappings in RVISC). The motion was seconded by Christoph Plutte. No objections, discussions or concerns were raised. Each voting member present then stated their vote. The following Voting Members voted in favor:
```

Author: Thank You! Suggestions were applied and comment is resolved.

- Celia Savidge
- Christoph Plutte
- JD Stefaniak

> - Omar Santos

Reviewer: Omar did not have voting rights at that time.
```suggestion
```

Author: Thank You! Suggestions were applied and comment is resolved.

- Sonny van Lingen suggested looking at real-world data points from large document collections.

#### Issue [#1348](https://github.com/oasis-tcs/csaf/issues/1348) - Remove leap second conversion rule

- Thomas Schmidt presented Issue #1348, which proposes to remove the leap second conversion rule as it is considered non-significant.
- Thomas Schmidt argued the leap second conversion rule is actively used by implementers as guidance for handling leap seconds (which are not well supported by many libraries), both for conversion and for handling such situations in CSAF 2.0.
- A motion was presented by Thomas Schmidt to reject the suggestion in Issue #1348 and keep the leap second conversion rules in CSAF 2.1. The motion was seconded by Sergii Demianchuk, who recalled the original discussion in April 2025 at the in-person meeting and reaffirmed support. No objections, discussions or concerns were raised. The motion carries.

### Outstanding Items

- Thomas Schmidt requested volunteers to write missing meeting minutes; the corresponding open issues are labelled `meeting_minutes`. Recordings can be organised where available.

> - Thomas Schmidt asked for help organising the CSAF Community Days (preliminary dates: November 16-17, 2026). Open roles include program committee, OASIS liaison and overall organiser. OASIS Open will be the official organiser of the event (as last year). Sonny van Lingen agreed to discuss further with Thomas offline.

Reviewer: ```suggestion
- Thomas Schmidt asked for help organizing the CSAF Community Days (preliminary dates: November 16-17, 2026). Open roles include program committee, OASIS liaison and overall organizer. OASIS Open will be the official organizer of the event (as last year). Sonny van Lingen agreed to discuss further with Thomas offline.
```

Author: Thank You! Suggestions were applied and comment is resolved.
